proposals/drafts/campaign_finance_filings.rst

```diff
@@ -162,7 +158,7 @@ actions
         "revocation" - allows for consolidating different jurisdictional
         amendment schemes into standard types.
 
-    inciting_person
+    inciter
```

Comment:
I guess so. Though probably most of the time, the "inciting organization" would be the Committee, so this field wouldn't be useful in those contexts. Much/most of the time, I think it's a person who's named on the filings, and I've found those to be useful for various purposes. Maybe it should be repeated and optional; I've updated this field and also responsible_person to be optional/repeated. (responsible_person may be duplicative with the inciter of each action).